cmd/generator/main.go

```diff
@@ -120,28 +106,21 @@ var generatorCmd = &cobra.Command{
 		for _, repositoryName := range repositoryNames {
 			logger.Debugf("querying repository %s", repositoryName)
 
-			releasePlugins, err := getReleasePlugins(ctx, client, repositoryName, includePreRelease, existingPlugins)
+			var releasePlugins []*model.Plugin
```

Comment:
Can you elaborate on why this is now explicitly declared?

Reply:
It's because it would shadow the error shadowing in L56 otherwise. Ugly, I know :confused: